setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+exclude = venv/,digits/,plugins/
```

Comment:
If `LINT_CHECK` is added to an `allow_failures` section maybe we can include everything?